infra/build/functions/base_msan_builder.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+"""Cloud function to build base images on Google Cloud Builder."""
+
+import logging
+
+import google.auth
+from googleapiclient.discovery import build
+
+import build_base_images
+import build_msan_libs
+
+BASE_PROJECT = 'oss-fuzz-base'
+
+
+# pylint: disable=no-member
+def base_msan_builder(event, context):
```

Comment:
This is almost identical to base_images. Can just add it there, and factor the common bits out. Only things different here are the steps, images, and timeout. For simplicity, can just use 6hrs for base images too.